docs/clusterdefinition.md

```diff
@@ -24,6 +33,8 @@ Here are the valid values for the orchestrator types:
 3. `Swarm` - this represents the [Swarm orchestrator](swarm.md).
 4. `Swarm Mode` - this represents the [Swarm Mode orchestrator](swarmmode.md).
 
+For `KubernetesConfig.KubernetesImageBase` the default value is GCR (gcr.io/google_containers/), ACS engine will use this base URL and the default Kubernetes Docker image name to concat the full URL of the Docker image URL such as `gcr.io/google_containers/hyperkube-amd64:v1.5.3`. In some the location GCR may be blocked or unstable to access, by specifying this value can redirect to custom Docker image download URL.

```

Comment:
If KubernetesImageBase is specified then generated templates will use custom Docker image download URL.